modules/virt-creating-storage-class.adoc

```diff
@@ -9,17 +9,66 @@
 When you create a storage class, you set parameters that affect the
 dynamic provisioning of persistent volumes (PVs) that belong to that storage class.
 
-[IMPORTANT]
-====
-When using {VirtProductName} with {product-title} Container Storage, specify RBD block mode persistent volume claims (PVCs) when creating virtual machine disks. With virtual machine disks, RBD block mode volumes are more efficient and provide better performance than Ceph FS or RBD filesystem-mode PVCs.
+In order to use the host path provisioner (HPP) you must create an associated storage class for the CSI driver with the `storagePool` parameter.
 
-To specify RBD block mode PVCs, use the 'ocs-storagecluster-ceph-rbd' storage class and `VolumeMode: Block`.
+[NOTE]
 ====
+You cannot update a `StorageClass` object's parameters after you create it.
+====
+
+== Creating a storage class for the CSI driver with the `storagePool` parameter
```

Comment:
Same here re: removing backticks from `storagePool` in the heading. Also needs a unique ID/anchor added.

Reply:
Done.